sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdRequestManager.java

```diff
@@ -548,10 +546,10 @@ boolean hasRntbdContext() {
         return this.contextFuture.getNow(null) != null;
     }
 
-    boolean isServiceable(final int demand, boolean ignoreMaxRequestPerChannel) {
+    boolean isServiceable(final int demand) {
```

Comment:
why don't we need `ignoreMaxRequestPerChannel` anymore?

Reply:
Discussed offline - not needed anymore.